* Do not override $(docdir) as part of the GNU conventions
* Do not install 'COPYING' and 'INSTALL'
* Install html files using html_ prefix and not doc_ prefix

--- a/Makefile.am
+++ b/Makefile.am
@@ -11,12 +11,11 @@
 AM_CXXFLAGS = -Wall -W -Wwrite-strings -Woverloaded-virtual -Wshadow
 endif
 
-docdir = $(prefix)/share/doc/$(PACKAGE)-$(VERSION)
 ## This is for HTML and other documentation you want to install.
 ## Add your documentation files (in doc/) in addition to these boilerplate
 ## Also add a TODO file if you have one
-dist_doc_DATA = AUTHORS COPYING ChangeLog INSTALL NEWS README README_windows.txt \
-   TODO \
+dist_doc_DATA = AUTHORS ChangeLog NEWS README README_windows.txt TODO
+dist_html_DATA = \
    doc/dense_hash_map.html			\
    doc/dense_hash_set.html			\
    doc/sparse_hash_map.html			\
